common/containers/Tabs/Swap/components/receivingAddress.js

```diff
@@ -19,6 +20,7 @@ export default class ReceivingAddress extends Component {
   };
 
   onClickPartTwoComplete = () => {
+    this.props.stopLoadBityRates();
```

Comment:
Good question. The existing site seems to just keep polling, so maybe its okay for us too? Do you have any ideas?